internal/kms/kms.go

```diff
@@ -436,38 +437,42 @@ func (k *Kms) RunKeyRevocation(ctx context.Context, keyRevocation *KeyRevocation
 		return errors.Wrap(ctx, err, op)
 	}
 
-	fmt.Println("Get rows affected by revocation")
-	select {
-	case <-time.After(3 * time.Second):
-	case <-ctx.Done():
-		fmt.Println("canceled")
-		return ctx.Err()
-	}
-
-	fmt.Println("Update progress for each row revoked")
-	select {
-	case <-time.After(3 * time.Second):
-	case <-ctx.Done():
-		fmt.Println("canceled")
-		return ctx.Err()
+	switch {
+	case strings.HasPrefix(keyRevocation.KeyId, "krkv"):
+		// Root key, rewrap keys in scope
+		// TODO: A better way to get scope_id from key version private id
+		// kms doesn't export the type.
```

Comment:
I'm leaning towards exporting `RootKeyVersion` from extras/kms/v2 to be able to scan this easily.

Reply:
I would hesitate to export this type... since it's not exported in any of that pkgs operations.   You could just Kms.ListKeys(...) which lists the keys for a scope and then find the one with the matching id in the list?